src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -238,7 +247,8 @@ export default class TableComponent extends ComponentBase implements IComponent,
 			this._table.setSelectedRows(this.selectedRows);
 		}
 
-		Object.keys(this._checkboxColumns).forEach(col => this.registerCheckboxPlugin(this._checkboxColumns[col]));
+		Object.keys(this._checkboxColumns).forEach(col => this.registerPlugins(col, this._checkboxColumns[col]));
```

Comment:
@llali I think this should work but please do check once that this continues to work for Schema compare.... thats the only place I know of where checkbox plugin is used.

Reply:
I did test schema compare to make sure I don't break radio button

---
In reply to: [456148917](https://github.com/microsoft/azuredatastudio/pull/10918#discussion_r456148917) [](ancestors = 456148917)